package.json

```diff
@@ -15,6 +15,7 @@
   },
   "license": "MPL-2.0",
   "dependencies": {
+    "debug": "^4.1.1",
```

Comment:
It does need to be a proper dep since it is included in the actual code and not just tests. I'm not concerned about this because its quite light, and this plugin will only run as part of the build process regardless. Also the fact that express and babel both ship with it as a prod dependency makes me feel like its fairly standard